src/domain/campaigns/campaign_details/CampaignDetails.module.scss

```diff
@@ -114,11 +153,32 @@
 .caption {
   font-weight: $bold;
 }
+.actionFooter {
+  text-align: center; 
+  margin-top: 50px;
+  p {
+    width: 50%;
+    margin-left: 25%;
+    font-size: 18px;
+  }
+  
+  
+}
+
+.actionIconContainer {
```

Comment:
Again I think this sass section is pretty flat and would probably benefit from some more nesting